aws-predictions/src/main/java/com/amplifyframework/predictions/aws/operation/AWSIdentifyOperation.java

```diff
@@ -29,6 +29,10 @@
 import java.util.Objects;
 import java.util.concurrent.ExecutorService;
 
+/**
+ *
+ * Operation that identifies objects within an image via AWS Translate, AWS Rekognition, or AWS Comprehend.
```

Comment:
Technically these are Amazon Translate, Amazon Rekognition, and Amazon Comprehend.